assets/questions/bomberman.js

```diff
@@ -124,8 +127,9 @@ globalData.questions['bomberman'] = {  // eslint-disable-line dot-notation
         "Does the result match what you expected? "
```

Comment:
This message doesn't seem consistent with the style (specifically having each open brace on its own line) of other buggyOutputTests. Please check the other files and make sure you're being consistent within the codebase's style.